packages/api/endpoints/granules.js

```diff
@@ -578,11 +579,27 @@ async function bulkOperations(req, res) {
  */
 async function bulkDelete(req, res) {
   const payload = req.body;
-
   if (payload.forceRemoveFromCmr && !isBoolean(payload.forceRemoveFromCmr)) {
     return res.boom.badRequest('forceRemoveFromCmr must be a boolean value');
   }
 
+  const concurrency = payload.concurrency ? Number(payload.concurrency) : 10;
+  const maxDbConnections = payload.MaxDbConnections
+    ? Number(payload.MaxDbConnections)
+    : concurrency;
+
+  try {
+    ['concurrency', 'maxDbConnection'].forEach((key) => {
+      if (payload[key] && !Number.isInteger(payload[key])) {
```

Comment:
If we're making sure that these are integers, why are we sending them through `Number()` above?